jobs/pr_gate/test_config/FIT/stack_config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_comment":{
+    "bmc": "appliance bmc address",
+    "control": "control switch admin address",
+    "data": "data switch admin address",
+    "hyper": "hypervisor admin address",
+    "rackhd_host": "appliance admin address",
+    "ovamac": "appliance OVA MAC address for ESXi deployments",
+    "pdu": "PDU admin address",
+    "type": "deployment type: esxi, docker, linux, or vagrant"
+  },
+  "local_run":{
```

Comment:
Can't we add this instead into rackhd/test/config/stack_config.json?  
Or if you really need a new directory, create the whole directory for FIT under rackhd/test/config/FIT/stack_config.json?
Then to run - run_tests.py -stack docker_local_run -config rackd/test/config/FIT